src/packages/@ncigdc/containers/ConsequencesTable.js

```diff
@@ -77,7 +77,12 @@ const ConsequencesTableComponent = compose(
           consequence: transcript.consequence_type,
           coding_dna_change: transcript.annotation.hgvsc,
           strand: transcript.gene.gene_strand
-            ? strandIconMap[transcript.gene.gene_strand.toString(10)]
+            ? <span>
+                {strandIconMap[transcript.gene.gene_strand.toString(10)]}
+                <ForTsvExport>
```

Comment:
That seems like it would require more work to support column sorting and column selection.

Would also be made obsolete with the TSV export endpoint that's in the backlog

Also not a fan of "if you're going to be making [small change], then you might as well make [larger change]". An easy way to have a PR snowball out of scope